不知道为什么用go call python
会有一个cache错误.
所以只能python自己开一个server了.